src/core/src/energy/functions.rs

```diff
@@ -487,4 +531,18 @@ mod tests {
         let e1 = born.energy(4.0 + EPS);
         assert_relative_eq!((e0 - e1) / EPS, born.force(4.0), epsilon=1e-6);
     }
+  #[test]
+    fn MorsePotential() {
```

Comment:
Could you rename this function to something like `morse` (to be in line with `lj`, `harmonice`, etc.)? Function names should always be lower case. That may sound picky, but it makes the code more readable 😄